units_generator/src/main/resources/data/units_constants.json

```diff
@@ -184,7 +243,7 @@
 						"denominators" : ["seconds", "seconds"]
 					},
 					"value" : 6.67408e-11,
-					"description" : "a constant reguarding the force between to objects, depending on their masses and distance"
+					"description" : "a constant regarding the force between to objects, depending on their masses and distance"
```

Comment:
"two objects", not "to objects". my bad.

Reply:
will fix it